pandas/tests/plotting/test_misc.py

```diff
@@ -313,3 +313,14 @@ def test_get_standard_colors_random_seed(self):
         color1 = _get_standard_colors(1, color_type='random')
         color2 = _get_standard_colors(1, color_type='random')
         assert color1 == color2
+
+    def test_get_standard_colors_no_appending(self):
+        # GH20726
+
+        # Make sure not to add more colors so that matplotlib can cycle
+        # correctly.
```

Comment:
ideally you can also test the example in the OP (as well)

Reply:
Added.